viewing_party/party.py

```diff
@@ -0,0 +1,193 @@
+# Wave 1
+def create_movie(title, genre, rating):
+    """
+    This function returns a dictionary that gives information about a
+    movie's title, genre and rating when all 3 parameters are truthy. 
+    Otherwise, the function returns None.
+    """
+    new_movie = {}
+    if title and genre and rating:
+        new_movie["title"] = "Title A"
+        new_movie["genre"] = "Horror"
+        new_movie["rating"] = 3.5
+        return new_movie
```

Comment:
be careful here with "hardcoding" values. Hardcoding means that you are using values that can't be changed, like "Title A", etc. What happens if we want to use this function several time to create more than one movie?

There's where the parameters on line 2 come into play:

```suggestion
    new_movie = {}
    if title and genre and rating:
        new_movie["title"] = title
        new_movie["genre"] = genre
        new_movie["rating"] = rating
        return new_movie
```

now, no matter what arguments are passed into the function `create_movie("Babe", "Drama", 3.8)` or `create_movie("Babe: Pig in the City", "Drama", 2.3)` this function can make _different_ movies, not just one with hardcoded data